app/controllers/api/v1/labware_types_controller.rb

```diff
@@ -7,12 +7,12 @@ class Api::V1::LabwareTypesController < Api::V1::ApplicationController
   def index
     @labware_types = LabwareType.all
 
-    render json: @labware_types
+    render json: @labware_types, include: [:layout]
   end
 
   # GET /labware_types/1
   def show
-    render json: @labware_type
+    render json: @labware_type, include: [:layout]
```

Comment:
We should probably separate that include: array into a function as it should be the same for each method.